### DISCUSSION TITLE
Cannot monitor mimir since /metrics endpoint is not available

### DISCUSSION THREAD

**Discussion by danielmarom78** [grafana/mimir#6643] [Help and support]
I installed mimir on openshift and I don't have enough permissions to install the Grafana agent for monitoring so I didn't enable metamonitoring.

I tried installing prometheus as a collector and setting it to collect from 
<gateway exposed rout url > /metrics

Trying to access this URL reterns 404

**Comment by dimitarvdimitrov** [comment]
which service are you using to access /metrics? The nginx pod that comes with the helm chart does not expose /metrics. However, all mimir and memcached pods serve /metrics on their HTTP port (8080 by default)

**Reply by danielmarom78** [reply]
The gateway.
Which one I need to access to get the whole cluster's metrics?

**Reply by dimitarvdimitrov** [reply]
> The gateway.

I assume this is the gateway deployment, which is nginx 👍 

in order to scrape the cluster's metrics you need to scrape each individual pod. Each Mimir pods exposes /metrics on 8080. You should configure the prometheus instance to discover and scrape those pods.
